plugins/dfu/dfu-target.c

```diff
@@ -730,6 +730,13 @@ dfu_target_download_chunk (DfuTarget *target, guint16 index, GBytes *bytes, GErr
 		g_usleep (dfu_device_get_download_timeout (priv->device) * 1000);
 	}
 
+	/* If the status returned by the device is dfuMANIFEST
+	 * detach has occurred, expect the device to disconnect */
+	if (dfu_device_get_state(priv->device) == DFU_STATE_DFU_MANIFEST) {
```

Comment:
Should this maybe also look for `attach-into-application`?  Normally I think this is not the expected behavior and should raise some red flags.

Reply:
I think i should check the attribute bitManifestationTolerant, as stated in the documentation.

https://www.usb.org/sites/default/files/DFU_1.1.pdf
> Device that cannot communicate via the USB after the Manifestation phase indicated this limitation to the host by clearing bmAttributes bit bitManifestationTolerant. 

what do you think?
unfortunately i can't check regression, because i don't such device.